src/models/goal.js

```diff
@@ -81,7 +83,7 @@ export default (sequelize, DataTypes) => {
       default: false,
     },
     isRttapa: {
-      type: DataTypes.ENUM(['Yes', 'No']),
+      type: DataTypes.ENUM(RTTAPA_ENUM),
```

Comment:
Please no, please just change this to an bool.

Reply:
it's been ['Yes', 'No'] for months - doesn't make sense to expand the scope of this PR to change that IMO (I'd have to dig into loads of unrelated code)

Reply:
ok

Reply:
https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1548